pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -700,7 +703,7 @@ func getVMIMigrationDataSize(vmi *v1.VirtualMachineInstance) int64 {
 func liveMigrationMonitor(vmi *v1.VirtualMachineInstance, l *LibvirtDomainManager, options *cmdclient.MigrationOptions, migrationErr chan error) {
 
 	logger := log.Log.Object(vmi)
-
+	migrateInfo := stats.DomainJobInfo{}
```

Comment:
you can just initialize this with an empty struct
and empty it when the monitor exists 
so something like
defer func() {
    l.migrateInfoStats = &stats.DomainJobInfo{
        DataProcessed 0,
        DataRemaining 0,
        MemDirtyRate 0,
        TransferRate 0
}

Reply:
Done